wiki/About/pl.md

```diff
@@ -0,0 +1,5 @@
+o osu!wiki
+=======
+
+osu!wiki to projekt kierowany przez społeczność, który pozwala zrozumieć użytkownikom osu! zasady i mechanikę gry w skuteczny sposób.
```

Comment:
+osu!wiki to projekt prowadzony przez społeczność, który ma za zadanie pomóc użytkownikom osu! skutecznie zrozumieć zasady i mechanikę gry.